### PR TITLE
Fix 189: add stale bot

### LINKED ISSUE #189: Add a bot for stale/old issue
- [ ] bot stale issues
- [x] pinned message in Github issues page

### PR DESCRIPTION
**Pull request by lvancraen**
Fix #189

### CODE REVIEW

**Comment by lvancraen** [comment]
@julienblin not sure if I need to create a changeset for this. Let me know. Also, I'm not a pro with github actions. Do you know an easy way to test that it works?

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          include-only-assigned: true
+          stale-issue-message: "This issue is stale because it has been open 15 days with no activity. Remove stale label or comment or this will be closed in 5 days."
+          stale-pr-message: "This PR is stale because it has been open 15 days with no activity."
+          close-issue-message: "This issue was closed because it has been stale for 5 days with no activity."
+          days-before-stale: 15
+          days-before-close: 5
```
Comment (julienblin) [inline .github/workflows/stale.yml:18]: I would recommend we use another 15 days before closing, given our current rate of action.

**Review by julienblin — APPROVED** [review]
Thank you!
I don't think you need a changeset since you are not touching any packages - the build is fine ATM.

As for testing GH actions, it's always a PITA; there is always [Act](https://github.com/nektos/act) to try to run them locally, but in this case I would be OK to just merge it and see how it goes!

**Comment by julienblin** [comment]
Closing this because of https://github.com/bonfhir/bonfhir/issues/189#issuecomment-1927102179

**Pull request closed without merging**